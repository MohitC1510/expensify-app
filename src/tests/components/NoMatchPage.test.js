import React from 'react';
import {shallow} from 'enzyme';
import NoMatchPage from '../../components/NoMatchPage';

test('should render not found page correctly', ()=>{
    const wrapper= shallow(<NoMatchPage />);
    expect(wrapper).toMatchSnapshot();
})