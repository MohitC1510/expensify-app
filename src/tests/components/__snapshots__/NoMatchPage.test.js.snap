// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found page correctly 1`] = `
<div>
  <p>
    404 not found!-
    <Link
      to="/"
    >
      Go home
    </Link>
  </p>
</div>
`;
